handlers/main.yml

```diff
@@ -3,7 +3,13 @@
 - name: restart_gitlab_runner
   service: name=gitlab-runner state={{ gitlab_runner_restart_state }}
```

Comment:
kind of unrelated to your patch:

wouldn't it be possible to join the multiple handlers into one based on the service name defined somewhere else?

Reply:
Yes I think this a good suggestion. I did not know if the author wanted such a modification to the entire project.